### ISSUE TITLE
npm install?

### ISSUE THREAD

**Issue by ORESoftware** [samrose3/ng-json-viewer#3]
Hey there!

This looks like what I am looking for, I have two questions

1. is there a way to install this with NPM or a CDN?

2. We are on AngularJS 1.6, do you think there is a better module than this one that does something similar?

**Comment by ORESoftware** [comment]
Also I have this:

```html
     <div class="row">
           <div style="height: 500px" class="form-group">
             <jv-json-viewer data="gitLogJSON" editor="false"></jv-json-viewer>
           </div>
     </div>
```

if gitLogJSON is an array or a stringified array, either way nothing is showing up, kinda weird

**Comment by sam-b-rose** [comment]
Hi @ORESoftware!

1. I can look into registering this as an NPM package if you would like to use this module for your project. Since I am not actively maintaining this currently, it might be best to fork the project and register an updated version. If you do get it working with your code, lmk and I will be happy to get this version into the npm registry.
2. I'm not sure what the compatibility is with Angular 1.6. I haven't touched this project in a long time ever since moving to other frameworks 😞 I haven't researched it, but I would assume there are alternative JSON viewer components available. Again, this one is rather old.

Make sure your `div` with the fixed height also is `position: relative`. The viewer is absolutely positioned and will need a relative parent as a reference. 👍

**Comment by ORESoftware** [comment]
Yeah thanks, let me see if it works with `position: relative` if not, might have to find something else. Surprisingly hard to google for "JSON AngularJS view/display", not that much comes up.
